diff --git a/dist/lint.js b/dist/lint.js
index a85201a13852e7912fb65b643861d988ccbc7515..4db0e61d7f64079906a86b06ce630373b2e4bf36 100644
--- a/dist/lint.js
+++ b/dist/lint.js
@@ -15264,7 +15264,8 @@ function setRegisteredRules(rules) {
 const neverRunBeforeHook = () => !1;
 async function loadPlugin(url, pluginName, pluginNameIsAlias, workspaceUri) {
 	try {
-		let plugin = (await import(url)).default;
+		let p = await import(url);
+		let plugin = p.default ?? p;
 		return JSONStringify({ Success: registerPlugin(plugin, pluginName, pluginNameIsAlias, workspaceUri) });
 	} catch (err) {
 		return JSONStringify({ Failure: getErrorMessage(err) });
