diff --git a/example/App.js b/example/App.js
index c9cd14d..f63758d 100644
--- a/example/App.js
+++ b/example/App.js
@@ -2,6 +2,7 @@
 import React, { useCallback, useMemo, useState } from "react";
 import {
   NativeModules,
+  SafeAreaView,
   ScrollView,
   StatusBar,
   StyleSheet,
@@ -10,7 +11,6 @@ import {
   useColorScheme,
   View,
 } from "react-native";
-import { SafeAreaProvider, SafeAreaView } from "react-native-safe-area-context";
 // @ts-expect-error
 import { version as coreVersion } from "react-native/Libraries/Core/ReactNativeVersion";
 import { Colors, Header } from "react-native/Libraries/NewAppScreen";
@@ -169,7 +169,6 @@ const App = ({ concurrentRoot }) => {
   );
 
   return (
-    <SafeAreaProvider>
       <SafeAreaView style={styles.body}>
         <StatusBar barStyle={isDarkMode ? "light-content" : "dark-content"} />
         <ScrollView
@@ -192,7 +191,6 @@ const App = ({ concurrentRoot }) => {
           </View>
         </ScrollView>
       </SafeAreaView>
-    </SafeAreaProvider>
   );
 };
 
diff --git a/example/package.json b/example/package.json
index a51d4cf..512b9c6 100644
--- a/example/package.json
+++ b/example/package.json
@@ -32,7 +32,6 @@
     "react": "17.0.2",
     "react-native": "^0.68.2",
     "react-native-macos": "^0.68.3",
-    "react-native-safe-area-context": "^4.3.4",
     "react-native-test-app": "workspace:.",
     "react-native-windows": "^0.68.8"
   },
