diff --git a/example/App.tsx b/example/App.tsx
index edc2259..7168465 100644
--- a/example/App.tsx
+++ b/example/App.tsx
@@ -2,6 +2,7 @@ import React, { useCallback, useMemo, useState } from "react";
 import type { LayoutChangeEvent, NativeSyntheticEvent } from "react-native";
 import {
   NativeModules,
+  SafeAreaView,
   ScrollView,
   StatusBar,
   StyleSheet,
@@ -10,7 +11,6 @@ import {
   useColorScheme,
   View,
 } from "react-native";
-import { SafeAreaProvider, SafeAreaView } from "react-native-safe-area-context";
 // @ts-expect-error
 import { version as coreVersion } from "react-native/Libraries/Core/ReactNativeVersion";
 import { Colors, Header } from "react-native/Libraries/NewAppScreen";
@@ -179,7 +179,6 @@ function App({ concurrentRoot }: AppProps): React.ReactElement<AppProps> {
   );
 
   return (
-    <SafeAreaProvider>
       <SafeAreaView style={styles.body}>
         <StatusBar barStyle={isDarkMode ? "light-content" : "dark-content"} />
         <ScrollView
@@ -202,7 +201,6 @@ function App({ concurrentRoot }: AppProps): React.ReactElement<AppProps> {
           </View>
         </ScrollView>
       </SafeAreaView>
-    </SafeAreaProvider>
   );
 }
 
