diff --git a/example/App.tsx b/example/App.tsx
index 35fea8c..56c1c0c 100644
--- a/example/App.tsx
+++ b/example/App.tsx
@@ -2,6 +2,7 @@ import React, { useCallback, useMemo, useState } from "react";
 import type { NativeSyntheticEvent } from "react-native";
 import {
   NativeModules,
+  SafeAreaView,
   ScrollView,
   StatusBar,
   StyleSheet,
@@ -10,7 +11,6 @@ import {
   useColorScheme,
   View,
 } from "react-native";
-import { SafeAreaProvider, SafeAreaView } from "react-native-safe-area-context";
 // @ts-expect-error no type definitions available
 import { version as coreVersion } from "react-native/Libraries/Core/ReactNativeVersion";
 import { Colors, Header } from "react-native/Libraries/NewAppScreen";
@@ -190,7 +190,6 @@ export function App({
   const [isFabric, setIsFabric] = useIsFabricComponent();
 
   return (
-    <SafeAreaProvider>
       <SafeAreaView style={styles.body}>
         <StatusBar barStyle={isDarkMode ? "light-content" : "dark-content"} />
         <ScrollView
@@ -213,6 +212,5 @@ export function App({
           </View>
         </ScrollView>
       </SafeAreaView>
-    </SafeAreaProvider>
   );
 }
diff --git a/example/package.json b/example/package.json
index fe0f522..7fb0f7e 100644
--- a/example/package.json
+++ b/example/package.json
@@ -21,7 +21,6 @@
     "react": "18.2.0",
     "react-native": "^0.73.0",
     "react-native-macos": "^0.73.0",
-    "react-native-safe-area-context": "~4.8.2",
     "react-native-windows": "^0.73.0"
   },
   "devDependencies": {
